docs/guides/migrations/guide.rst

```diff
@@ -0,0 +1,1298 @@
+made, and will fail otherwise.
+
+A non-interactive migration will fail if we make changes to our schema 
+that are ambiguous. Let's see if we can make a non-interactive migration 
+fail by doing just that. Delete the most recent ``.edgeql`` migration 
+file again, and change the schema to this:
+
+.. code-block:: sdl
+
+    module default {
+      type User {
+        nam: Name;
+      }
+      scalar type Name extending str;
+    }
+
+The only difference from the current schema is that we would like 
+to change the property name ``name`` to ``nam``.
+
+But this time EdgeDB isn't sure what change we wanted to make. Did 
+we intend to:
+
+- Change ``name`` to ``nam`` and keep the existing data?
+- Drop ``name`` and create a new property called ``nam``?
+- Do something else?
+
+So this time the non-interactive migration will fail, with some pretty 
+nice output:
+
+.. code-block:: edgeql-repl
+
+    db> \migration create --non-interactive
+    EdgeDB intended to apply the following migration:
+        ALTER TYPE default::User {
+            ALTER PROPERTY name {
+                RENAME TO nam;
+            };
+        };
+    But confidence is 0.67, below minimum threshold of 0.99999
+    Error executing command: EdgeDB is unable to make a decision.
+
+    Please run in interactive mode to confirm changes, or use
+    `--allow-unsafe`
+
+As the output suggests, you can add ``allow-unsafe`` to a non-interactive 
+migration if you truly want to push the suggestions through regardless 
+of the migration tool's confidence. But it's more likely in this case 
+that you would like to interact with the CLI's questions to help it 
+make a decision. For example, if we had intended to drop the property 
+``name`` and create a new property ``nam``, we would simply answer 
+``n`` when it asks us if we intended to *rename* the property. It 
+then confirms that we are altering the ``User`` type, and finishes 
+the migration script.
+
+.. code-block:: edgeql-repl
+
+    db> \migration create
+    did you rename property 'name' of object type 'default::User' 
+    to 'nam'? [y,n,l,c,b,s,q,?]
+    > n
+    did you alter object type 'default::User'? [y,n,l,c,b,s,q,?]
+    > y
+
+Afterwards, you can go into the ``.edgeql`` file that was just created 
+to confirm that these were the changes we wanted to make. It will 
+look like this:
+
+.. code-block::
+
+    CREATE MIGRATION m15hu2pbez5od7fe3shlxwcprbqhvctnfavadccjgjszboy26grgka
+        ONTO m17m6qjjhtslfkqojvjb4g2vqtzasv5mlbtrqbp6mhwlzv57p5f2uq
+    {
+      ALTER TYPE default::User {
+        CREATE PROPERTY nam: default::Name;
+        DROP PROPERTY name;
+      };
+    };
+
+.. note::
+
+    See the section below on migration hashes if you are curious about 
+    how migrations are named)
+
+So the ``User`` type was altered via creating a new property, and 
+dropping the old one. If that is what we wanted, then we can now type 
+``edgedb migrate`` to complete the migration.
```

Comment:
```suggestion
``\migrate`` in the REPL or ``edgedb migrate`` at the command line to complete the migration.
```
Leading with the REPL command since this is what you used in the previous step.